contracts/src/spaces/entitlements/rule/RuleEntitlement.sol

```diff
@@ -166,28 +161,14 @@ contract RuleEntitlement is
     }
 
     Entitlement storage entitlement = entitlementsByRoleId[roleId];
```

Comment:
Do we want to overwrite existing `Entitlement` or revert if the `roleId` has been written?

Reply:
Overwrite so edit doesn't require a remove and a set call